code/game/mecha/mecha.dm

```diff
@@ -110,7 +110,6 @@
 	//Action vars
 	var/thrusters_active = FALSE
 	var/defence_mode = FALSE

```

Comment:
I meant this variable and the var in line 101. Probably out of scope of this PR because that would mean that you have to refactor mech UI and the simplemob mech pilot

Reply:
Yeah, I tried to move defence_mode to durand.dm, but then mecha_topic.dm throws a fit about it.